generator/generator.go

```diff
@@ -39,6 +39,7 @@ type Generator struct {
 	lowercaseLookup   bool
 	caseInsensitive   bool
 	marshal           bool
+	unknownCase       bool
```

Comment:
```suggestion
	unknownCase       string
	unknownCaseValue string
	unknownCaseSuffix string
```

I would make this as customizable as possible here. Allow setting a default value, and the suffix. For example in re:cap BE, the value is usually `NOT_SET`.

(We can't just use `unknownCase string`, in case someone wants to use an empty string as default).

Reply:
I thought about this, but was wondering how to have a different syntax between renaming and setting the default case:

```go
type Gender string

// This renames between type and value:
// ENUM(male, female, unknown=not_set)

// Maybe something like this?
// ENUM(male, female) default=not_set
```

This is a bit more work because we need to have extra parsing but doable

Reply:
You could just add it to the CLI args for now.

* `unknownCase:bool` -> enable if true
* `unknownCaseValue:*string` -> set default value if not nil
* `unknownCaseSuffix:*string` -> replace the `Unknown` suffix if not nil

Actually, the args would have to be `*string`, as `""` would be valid use cases.

Reply:
But anyway don't let me drag this out too much, we can also change this later.

Reply:
The CLI args apply for all enums defined in the file though, so I think if we have customization of the value and suffix, I think we should do it at the enum level 🤔

Reply:
Right, that way it's more flexible and the `//go generate ...` trigger can be the same across files.